sycl/plugins/hip/pi_hip.cpp

```diff
@@ -980,7 +980,7 @@ pi_result hip_piDeviceGetInfo(pi_device device, pi_device_info param_name,
                    PI_DEVICE_TYPE_GPU);
   }
   case PI_DEVICE_INFO_VENDOR_ID: {
-    return getInfo(param_value_size, param_value, param_value_size_ret, 4318u);
+    return getInfo(param_value_size, param_value, param_value_size_ret, 4098u);
```

Comment:
Out of curiosity, what vendor ID is returned when DPC++ was built with HIP NVIDIA support?

Reply:
Thank you for your suggestion. I don't have a platform for the test, but I added the following:
  
 case PI_DEVICE_INFO_VENDOR_ID: {
#if defined(__HIP_PLATFORM_AMD__)
    pi_uint32 vendor_id = 4098u;
#elif defined(__HIP_PLATFORM_NVIDIA__)
    pi_uint32 vendor_id = 4318u;
#else
    pi_uint32 vendor_id = 0u;
#endif
    return getInfo(param_value_size, param_value, param_value_size_ret, vendor_id);
  }

Reply:
@zjin-lcf - Are you planning on incorporating the changes you mention here?

Reply:
I checked the box 'Allow edits and access to secrets by maintainers', so assumed that reviewers could incorporate the changes.

Reply:
Okay. I updated the codes for pull request.